### ISSUE TITLE
Why does `--dry-run` require `--allow-write`

### ISSUE THREAD

**Issue by anishkny** [hayd/deno-udd#93]
`--dry-run` option should not be touching the file system and hence not require `--allow-write` to run.

**Comment by hayd** [comment]
Was this fixed by #91 ? Which version of udd are you running?

**Comment by anishkny** [comment]
Oh whoops you are right. It was resolving to my local cached older version. Is this expected behavior?

```bash
deno run https://deno.land/x/udd/main.ts --version
0.7.5

deno run https://deno.land/x/udd@0.8.1/main.ts --version
0.8.1
```

I thought `deno.land/x/udd/main.ts` should resolve to the latest but guess I was wrong. Could not find this in the manual.

In any case, closing this issue. Thanks for all the great work!
